drawee/src/main/java/com/facebook/drawee/drawable/RoundedCornersDrawable.java

```diff
@@ -162,17 +205,20 @@ private void updatePath() {
   @Override
   public void draw(Canvas canvas) {
     Rect bounds = getBounds();
+    int saveCount = canvas.save();
```

Comment:
Better to have this where it was before. If necessary add another `canvas.save` in the other case.